packages/util-package-finder/README.md

```diff
@@ -0,0 +1,157 @@
+# Package Finder
+
+[![NPM version][badge-npm]][info-npm]
+[![Node version][badge-node]][info-node]
+![LGPL 3.0 licensed][badge-license]
+
+Get a list of packages and available versions from within an NPM scope. Returns the following information about a package:
+
+* `name` the package name on NPM
+* `latest` the latest version of the package
+* `versions` array of all versions of the package
+* `status` the status based on the latest version _(see below)_
+* `description` the package description
+* `npm` a link to the package on NPM
+
+### Status
+
+The status of a package is evaluated by checking the latest version and assigning one of the following:
+
+| Status | Rule |
+| --- | --- |
+| production | `>= 1.0.0` |
+| development | `>= 0.1.0`, `< 1.0.0` |
+| experimental | `>= 0.0.1`, `< 0.1.0` |
```

Comment:
Should there be a `unpublished` status for `< 0.0.1` so that the `status` field isn't undefined?

Reply:
if it's unpublished it won't be on NPM, so won't ever appear in this list.